lib/dash/dash_parser.js

```diff
@@ -122,6 +123,16 @@ shaka.dash.DashParser = class {
 
     /** @private {shaka.util.Timer} */
     this.updateTimer_ = new shaka.util.Timer(() => {
+      if (this.mediaElement_ && !this.config_.continueLoadingWhenPaused) {
+        const onTimeUpdate = () => this.onUpdate_();
+        this.eventManager_.unlisten(
```

Comment:
This is supposed to keep the listenOnce from ticking multiple times if the update timer is called multiple times before a `timeupdate` event occurs, right?

I'm not sure if this `unlisten` will work properly though. You have to pass in the same callback that was passed in the first time, right? And this function makes a fresh `onTimeUpdate` callback every time it is called.

You might have to define `onTimeUpdate` outside of this function... or just not provide that argument at all, since the `listener` argument on `unlisten` is optional (if not provided, it will remove all listeners for the event).

Reply:
Updated!